.github/workflows/stale_labeler.yaml

```diff
@@ -0,0 +1,19 @@
+name: 'Label inactive PRs as stale'
+
+on:
+  schedule:
+    - cron: '0 11 * * *'  # runs at 11am UTC every day => early morning in US
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8  # https://github.com/marketplace/actions/close-stale-issues
+        with:
+          stale-pr-label: 'stale'
+          stale-pr-message: "This PR is being marked as stale because there hasn't been activity in 14 days. It may be closed by a rosdistro maintainer. If this label isn't appropriate, you can remove the label and add the \"don't mark as stale\" label. You can find our community contributing guidelines here: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md."
+          days-before-pr-stale: 14
```

Comment:
The other concern I have is what was brought up in the meeting; i.e. that this won't add a new comment every two weeks.  Can you verify whether that is the case or not?

Reply:
The docs say this, which doesn't fully answer the question:

> If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart

It seems like if the `stale` label exists, it won't make a comment. It also filters out bots commenting on PRs, so it doesn't consider them.

I've been looking at the code and it looks 
https://github.com/actions/stale/blob/465330b7e2524a0d9b3430795c7970658106bf5c/src/classes/issues-processor.ts#L798-L802

I don't think that we'll see this commenting multiple times on a stale issue.